deploy/cloud/modules/serverless/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "ec_url" {
+  description = "Elastic Cloud URL"
+  type        = string
+}
+
+variable "ec_apikey" {
+  description = "Elastic Cloud API Key"
+  type        = string
+}
+
+variable "project_name" {
```

Comment:
Is it the serverless name?
maybe we can use [random](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet) to ease things

Reply:
We already get a random suffix from the caller.